conf/requirements.txt

```diff
@@ -12,5 +12,5 @@ pandas
 yapf
 git+https://github.com/SymbiFlow/fasm.git#egg=fasm
 git+https://github.com/SymbiFlow/xc-fasm.git#egg=xc-fasm
-git+https://github.com/SymbiFlow/edalize.git@symbiflow#egg=edalize
+git+https://github.com/antmicro/edalize.git@quicklogic#egg=edalize
```

Comment:
The edalize changes to allow having the quicklogic toolchain should probably be merged in the symbiflow branch of the edalize fork.